libs/application/templates/financial-aid/src/fields/Summary/Files.tsx

```diff
@@ -21,20 +22,26 @@ interface Props {
 const Files = ({
   route,
   goToScreen,
+  personalTaxReturn,
   taxFiles,
   incomeFiles,
   applicationId,
 }: Props) => {
   const { formatMessage } = useIntl()
   const [createSignedUrlMutation] = useMutation(CreateSignedUrlMutation)
-  const allFiles = taxFiles.concat(incomeFiles)
+
+  const allFiles =
+    personalTaxReturn === null
+      ? incomeFiles.concat(taxFiles)
+      : incomeFiles.concat(taxFiles).concat([personalTaxReturn])
 
   return (
     <SummaryBlock editAction={() => goToScreen?.(route)}>
       <Text fontWeight="semiBold" marginBottom={1}>
         {formatMessage(summaryForm.formInfo.filesTitle)}
       </Text>
       {allFiles &&
+        allFiles.length > 0 &&
```

Comment:
Do we need to check the length of allFiles? Could we remove this check and then if the length is 0 then the allFiles.map will just return nothing?